app.py

```diff
@@ -1,8 +1,10 @@
 import flask
 from authlib.integrations.flask_client import OAuth
 from api import create_app
+from flask_cors import CORS
 
 app = create_app()
+CORS(app)
```

Comment:
This is fine for now, but when we deploy the app we'll probably want to limit CORS requests to a whitelisted set of origins.

Reply:
I agree with @blancadesal . Also we have set up a proxy on the frontend to help with the CORS issue in the development stage, so I was wondering what could bring this up again.  

Then I did some digging and I realised this [line](https://github.com/wikimedia/toolhunt/blob/main/api/config.py#L26) that was added yesterday could be the culprit. When I added the SERVER_NAME to the .env.dev file and set the value to `flask-web:5000`, the 404 errors stopped and data was received. `flask-web` is the name of the container the backend is running and as such it would be the identifier for other containers in the network to access its resources.

I did this because of this error in the logs:

`/usr/local/lib/python3.9/site-packages/flask/app.py:2218: UserWarning: Current server name 'flask-web:5000' doesn't match configured server name 'localhost:5000'`.

So maybe this isn't needed for this instance..